content/en/dashboards/widgets/_index.md

```diff
@@ -162,8 +162,6 @@ Widgets not linked to global time show the data for their local time frame as ap
 
 ## Copy and paste widgets
 
-<div class="alert alert-warning">You must have <a href="https://docs.datadoghq.com/account_management/rbac/permissions/#dashboards"><code>dashboard_public_share</code> permissions</a> and enable <a href="https://app.datadoghq.com/organization-settings/public-sharing/settings"><strong>Static Public Data Sharing</strong></a> in your Organization Settings to use this feature.</div>
```

Comment:
The permission isn't needed but the flag mentioned should be enabled. Do we still want to remove the whole sentence?

Reply:
Ooooh! So leave "Enable <a href="https://app.datadoghq.com/organization-settings/public-sharing/settings"><strong>Static Public Data Sharing</strong></a> in your Organization Settings to use this feature"

Reply:
Yeup 👍